bot/exts/moderation/clean.py

```diff
@@ -615,6 +619,13 @@ async def clean_cancel(self, ctx: Context) -> None:
         await self._send_expiring_message(ctx, message)
         await self._delete_invocation(ctx)
 
+    @command()
+    async def purge(self, ctx: Context, users: Greedy[User], age: Optional[Union[Age, ISODateTime]] = None) -> None:
+        """Clean messages of users from all public channels up to a certain message age (10 minutes by default)."""
```

Comment:
Is this limit inclusive? It should be documented.

Reply:
Hmm yeah, I didn't consider it because it would only be relevant if someone took a message, deciphered its exact age from the ID, and used it with the expectation the message would be deleted. Added in 154cd04 for good measure.